packages/blockchain-link/src/workers/baseWebsocket.ts

```diff
@@ -88,6 +88,7 @@ export abstract class BaseWebsocket<T extends EventMap> extends TypedEmitter<T &
         if (ws.listenerCount('open') > 0) {
             ws.emit('error', new CustomError('websocket_timeout'));
             try {
+                ws.once('error', () => {}); // hack; ws throws uncaughtably when there's no error listener
```

Comment:
no problem with this solution but doesn't it affect runtime too?

Reply:
Well it probably does, but it shouldn't do anything more than add an (either first one or subsequent) empty listener to already failing websocket in `try` block with empty `catch`, so I'm pretty confident™ that it shoudn't break anything.

What I'm not so confident about is whether the `ws` bump itself isn't breaking anything else in runtime outside of tests, because it definitely can.